backend/package.json

```diff
@@ -10,6 +10,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "bcrypt": "^4.0.1",
```

Comment:
I think we should use `bcryptjs` rather than `bcrypt`, since installing native modules brings it's own host of problems.

I appreciate that it's a bit slower, but it's still insanely fast.